integr8ly/integreatly-operator
# intly-2930 fuse 7.4 update

update the operator to be fuse 7.4 compatible by copying the pull
secret to the fuse namespace. add an override option to the rhmi
installation custom resource.

fix integrations not being able to be created by setting the limit
to 0.

verification:
- have an existing openshift 4 cluster
- create an operatorsource pointing at the repo aidenkeating
- create an installation
- ensure fuse reconciles successfully
- ensure images used by fuse come from the redhat registry
- ensure fuse 7.4 is deployed
- ensure an integration can be created in fuse
